packages/app/src/app/pages/Sandbox/Editor/Workspace/index.tsx

```diff
@@ -1,20 +1,22 @@
 import VERSION from '@codesandbox/common/lib/version';
 import { observer } from 'mobx-react-lite';
 import React from 'react';
+//  Fix css prop types in styled-components (see https://github.com/DefinitelyTyped/DefinitelyTyped/issues/31245#issuecomment-463640878)
+import * as CSSProps from 'styled-components/cssprop'; // eslint-disable-line
```

Comment:
Is there a way to not do this? I also am not sure what the advantage of the `css` prop is over `style`.

Reply:
I believe the core difference between the `css` prop and the `style` prop is that `style` is in-lined on the element, and `css` compiles to a styled component (ie: a uniquely named class). That would mean they have different levels of specificity as well as in-lining the styles into the markup.

However, just as we don't want to bloat the rendered HTML with styles, we shouldn't be bloating our JSX with style definitions either, as it makes it more difficult to read the structure of a component. It's generally better if we avoid inline styles entirely and use `styled` instead to definite presentation attributes in an `elements.ts` file.

The only reason why we would need to import the cssprop type definition is if we wanted a component to accept inline styles from a parent.  The better option would be to wrap the component in a call to `styled` instead.

Reply:
I haven't touched @Saeris in this PR, but if you guys want I can convert the components with a `css` props to be a new styled component.

Reply:
Yeah I think when I went over this originally, I kept some things as they were because I wasn't confident it didn't have some other purpose. My vote is to do away with inline styles, so if you'd like to make the change go right ahead.

Reply:
I'm on it! 😉

Reply:
@CompuIves @Saeris Converted all `css` prop call to new components in a `elements.ts` file